estimationpy/fmu_utils/model.py

```diff
@@ -1280,15 +1277,12 @@ def __set_fmu__(self, fmu_file, result_handler, solver, atol, rtol, verbose):
             # set the number of states
             self.N_STATES = len(self.get_state())
             
-            # get the value references of the state variables
-            self.stateValueReferences = self.fmu.get_state_value_references()
```

Comment:
``get_state_value_references()`` not valid method for FMI 2.0 using PyFMI.  ``self.stateValueReferences`` never found to be used in rest of repository.